addons/spreadsheet/static/tests/lists/list_plugin.test.js

```diff
@@ -417,6 +490,80 @@ test("can edit list domain", async () => {
     expect(getCellValue(model, "B2")).toBe("");
 });
 
+test("can edit list sorting", async () => {
+    const { model } = await createSpreadsheetWithList({
+        columns: ["foo", "bar", "date", "probability", "pognon"],
+    });
+    const [listId] = model.getters.getListIds();
+    expect(model.getters.getListDefinition(listId).orderBy).toEqual([]);
+    expect(getEvaluatedGrid(model, "A1:E5")).toEqual([
```

Comment:
We usually prettify this kind of lines. For exemple: 
```
    // prettier-ignore
    expect(getEvaluatedGrid(model, "A1:D5", "42")).toEqual([
        [1,         "",             11,             11],
        [2,         "",             15,             15],
        [12,        10,             "",             10],
        [17,        "",             95,             95],
        ["Total",   10,             121,            131],
    ]);
    ```
    `// prettier-ignore` is important to avoid the tooling to override it.